### PR TITLE
tickets/DM-18134 Get pipeline working with ctio0m9 data

### PR DESCRIPTION
**Pull request by mfisherlevine**

### CODE REVIEW

File: config/processStar.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import os.path
+from lsst.utils import getPackageDir
+
+config.load(os.path.join(getPackageDir("obs_lsst"), "config", "auxTel", "auxTel.py"))
+
+# Configuration for processStarTask
+
+if hasattr(config, 'ccdKeys'):
```
Comment (parejkoj) [inline config/processStar.py]: Why do you need this `hasattr` here? When would the config not have that?

File: config/processStar.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+config.isr.doBias = False
+config.isr.doLinearize = False
+config.isr.doDark = False
+config.isr.doFlat = False
+config.isr.doDefect = False
+config.isr.doCrosstalk = False
+config.isr.doSaturationInterpolation = False
+
+config.dispersionDirection = 'x'
```
Comment (parejkoj) [inline config/processStar.py]: Add a comment as to why you set this.

File: config/processStar.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+if hasattr(config, 'ccdKeys'):
+    config.ccdKeys = ['detector', 'detectorName']
+
+config.isr.doBias = False
+config.isr.doLinearize = False
+config.isr.doDark = False
+config.isr.doFlat = False
+config.isr.doDefect = False
+config.isr.doCrosstalk = False
+config.isr.doSaturationInterpolation = False
```
Comment (parejkoj) [inline config/processStar.py]: Add a comment as to why these are all turned off.

File: config/processStar.py (hunk 1 above)
Comment (mfisherlevine) [reply config/processStar.py]: Agreed. This was cargo-culted and can be removed.

**Pull request merged**